File: LinkedList.java
public class LinkedList{
	static class Node{
		int data;
		Node next;
		Node(int data){
			this.data=data;
			this.next=null;
		}
		
	}
	static Node insert(Node head,int data){
			if (head==null){
				return new Node(data);
			}
			Node temp=head;
			while(temp.next!=null){
				temp=temp.next;
				//System.out.println("data="+temp.data);
			}
			temp.next=new Node(data);
		
			return head;
		}
		
		static void print(Node head){
			Node temp=head;
			while(temp!=null){
				System.out.println(temp.data);
				temp=temp.next;
			}
			
		}
		static Node delete(Node head,int count){
			int c=0;
			Node temp=head;
			Node seTemp=head;
			Node prev=head;
			while(temp!=null){
				temp=temp.next;
			if(c>=count){
				prev=seTemp;
				seTemp=seTemp.next;
				
			}
			c++;
			
			}
			
			prev.next=seTemp.next;
			seTemp=null;
			return head;
		}
		
	public static void main(String args[]){
		
		Node head=null;
		
		System.out.println("enter no of nodes");
		java.util.Scanner scan=new java.util.Scanner(System.in);
		int n=scan.nextInt();
		for(int i=0;i<n;i++){
			int a=scan.nextInt();
			head=insert(head,a);
		}
		print(head);
		System.out.println("delete node form end eneter count");
		int c=scan.nextInt();
		head=delete(head,c);
		print(head);
		
	}
}

File: cProgram/linkedlIst.c
#include<stdio.h>
struct Node{
	int data;
	struct  Node* next;
};
void printnode(struct Node * head){
	while(head!=NULL){
		printf("%d",head->data);
		head=head->next;
	}
}
int main(){
struct Node*head=NULL;
struct Node*third=NULL;
struct Node*second=NULL;
head=(struct Node*)malloc(sizeof(struct Node));
third=(struct Node*)malloc(sizeof(struct Node));
second=(struct Node*)malloc(sizeof(struct Node));

(*head).data=1;
	head->next=second;
	
	
	second->data=2;
	second->next=third;
	
	third->data=3;
	third->next=NULL;
	printnode(head);
return 0;	
}
